llvm/test/CodeGen/AArch64/vecreduce-add.ll

```diff
@@ -1434,37 +1434,21 @@ entry:
 }
 
 define zeroext i8 @add_v16i8_v16i8_acc(<16 x i8> %x, i8 %a) {
-; CHECK-SD-BASE-LABEL: add_v16i8_v16i8_acc:
```

Comment:
This change here seem unnecessary, but it gets added by the update_llc_test_checks.py. Those checks pass with or without this change, and the only thing changing looks to be the --check-prefix. Using different versions of update_llc_test_checks.py doesn't make a difference either.

Reply:
I think it's because multiple prefixes were supplied on the same RUN line and different versions of the script have chosen different a prefix.